tasks/51_abbreviated-name/abbreviated-name.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * abbreviated - Converts a fullname in abbreviated form.
+ *
+ * @example
+ * //return 'Verbovyi D. O.'
+ * abbreviated('Verbovyi Dmytro Oleksandrovych');
```

Comment:
See how it should be in JSDoc

Reply:
@vovcyan  i used this https://jsdoc.app/tags-example.html for writing example part